### ISSUE TITLE
[5.0] Dark Mode: <code> colour fails a11y

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#42072]
### Steps to reproduce the issue

1. Go to `administrator/index.php?option=com_config`
2. Click the "Toggle Inline Help" button (top right of toolbar)

or:

Inject `<code>I'm text</code>` somewhere on the page

### Actual result

The colour used for `<code>` doesn't meet the minimum contrast requirements:

![image](https://github.com/joomla/joomla-cms/assets/2019801/bfe69047-0a19-42a3-9e0a-6671dae7a3ef)
